devbox.yaml

```diff
@@ -3,7 +3,7 @@
 # Lives in its own namespace to keep itself a little further away from all
```

Comment:
This deployment ends up being helpful for running things like vm creations, but I think these could be run just as easily (if not better) using `hope run create-vm` or something like that. Right now, I have to constantly delete these pods whenever I make a change to the image, and it would be easier if this was just ephemeral.